src/editors/containers/LibraryContentEditor/data/requests.js

```diff
@@ -0,0 +1,97 @@
+import { StrictDict } from '../../../utils';
```

Comment:
A lot here seems not covered by tests. Probably warrants a `requests.test.js` file.